index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nylbert's Portfolio</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter">
+  </head>
+  <body>
+    <header>
+      <nav class="nav">
+        <div id="ham-menu">
+          <button id="ham-but" type="button" class="hamburger">
+            <span class="ham-top"></span>
+            <span class="ham-mid"></span>
+            <span class="ham-botom"></span>
+          </button>
+        </div>
+        <ul id="menu">
+          <il><a href="#portfolio">Portfolio</a></il>
+          <il><a href="#about">About</a></il>
+          <il><a href="#contact">Contact</a></il>
+        </ul>
+      </div>
+    </header>
+    <div id="center-div">
```

Comment:
- [ ] According to the project requirements, this should be a `section`; it'll also make more sense semantically.